app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,56 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+#!/bin/sh
+<%
+    headers = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+-%>
+
+# Rendered with following template parameters:
+<%= "#  - User: [#{@user.login}]" if @user %>
+<%= "#  - Organization: [#{@organization.name}]" if @organization %>
+<%= "#  - Location: [#{@location.name}]" if @location %>
+<%= "#  - Hostgroup: [#{@hostgroup.name}]" if @hostgroup %>
 
 if ! [ $(id -u) = 0 ]; then
     echo "Please run as root"
     exit 1
 fi
```

Comment:
This is only really used in the Katello part. I'd keep the root check inside the subscription manager code. Curl can run as non-root just fine.

Reply:
Actually this will be useful in later PRs, where we will be setting up SSH keys for root user & when we will install insights pkg.